manifest.json

```diff
@@ -153,34 +153,11 @@
       "images": [
         {
           "sharedTags": {
-            "1.0.7": {},
-            "1.0": {},
-            "lts": {}
```

Comment:
Seems strange that the lts tag is not being moved to 1.1 - at least as an undocumented tag.

Reply:
Good catch. I'll move it to 1.1 as well. We want to drop it eventually since 'lts' isn't a good tag to use.